README.md: Add Windows support and mention MetaMask issue (#12)
oasisprotocol/demo-starter: README.md

*clear your account's activity* each time or manually specify the correct
account nonce.

When click button Connect Wallet, in some case you will constantly get the error: 
**Uncaught (in promise) Error: [useEthereumStore] Request account failed!**, 

To resolve it, try to open MetaMask and manually connect site once time

![alt text](manually-connect-site-1.png)  ![alt text](manually-connect-site-2.png)

> 

Reviewer: I cannot reproduce this one and it seems as a MetaMask-specific error. I would either not put this into README.md at all or add a new `## Troubleshooting` at the end of the README.md with the description above.

Please place the two screenshots into the new docs/ folder.

Author: I added a new ## Troubleshooting at the end of the README.md with the description above & moved two screenshots into the new docs/ folder

```

Then copy the `dist` folder to a place of your `/my/public/path` location.

## Troubleshooting

When click button Connect Wallet, in some case you will constantly get the error: 

> **Uncaught (in promise) Error: [useEthereumStore] Request account failed!**, 

Reviewer: ````suggestion
`Uncaught (in promise) Error: [useEthereumStore] Request account failed!`
````

> To resolve it, try to open MetaMask and manually connect site once time

Reviewer: ```suggestion
To resolve it, try to open MetaMask and manually connect site:
```

> ![alt text](docs/manually-connect-site-1.png)  ![alt text](docs/manually-connect-site-2.png)

Reviewer: ```suggestion
![MetaMask: Connected sites](docs/manually-connect-site-1.png)
![MetaMask: Manually connect to the current site](docs/manually-connect-site-2.png)
```
